docs/topics/proc_viewing-the-booster-source-code-and-readme.adoc

```diff
@@ -10,11 +11,10 @@ One of the following:
 
 .Procedure
 
-. Use the {name-launcher} tool to generate your own version of the booster.
-. View the generated GitHub repository or download and extract the ZIP file that contains the booster source code.
+. Use the {name-launcher} tool to generate your own version of the example.
```

Comment:
Use the {name-launcher} tool to generate your own version of the example. --> Use the {name-launcher} tool to generate your own version of the example application.

Reply:
Updated and better. Thanks!

Reply:
Thanks.  Made this change in the parallel modules as well.